javagen/src/main/java/com/azure/autorest/util/CodeNamer.java

```diff
@@ -266,7 +266,7 @@ public static String getEnumMemberName(String name) {
                 int preWidthWrapAt = -1;
                 int postWidthWrapAt = -1;
                 for (int i = 0; i != whitespacePositions.size() - 1; i++) {
-                    if (whitespacePositions.get(i) <= width && whitespacePositions.get(i + 1) > width) {
+                    if (whitespacePositions.get(i + 1) > width) {
```

Comment:
Break at first place that NEXT position over the line limit.

Since there is `break` in this `if` block, no need to guard against 2nd time this happens.